Makefile

```diff
@@ -41,3 +37,8 @@ schools_raw.csv:
 		TO STDOUT WITH CSV HEADER' > build/$@
 		touch build/$@
 
+# Create the main csv file with human readble data from IPEDS
+process:
+		python decoder.py
```

Comment:
The python program should take the dependency file from STDIN and pipe the output to the target filename. Ask @hancush how to do this.